gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# tl;dr: Projects are coupled if they change each others config and ours are not
+org.gradle.parallel=true
```

Comment:
Just to speed up the build. As the link in the gradle.properties says, it is perfectly safe as long as your subprojects configuration don't rely on any other subprojects configuration.  If we notice any weirdness, it is simple to turn this off.